src/data-workspace/use-data-value-mutation/use-upload-data-value-file-mutation.js

```diff
@@ -6,7 +6,15 @@ import useMutationFn from './use-mutation-fn.js'
 export const UPLOAD_FILE_MUTATION = {
     resource: 'dataValues/file',
     type: 'create',
-    data: (data) => data,
+    data: ({ co, de, ds, ou, pe, value, file }) => ({
```

Comment:
again would be nice not to have these short forms as the input for the mutations, but probably should be tackled separately

Reply:
https://dhis2.atlassian.net/browse/TECH-1293